3.6.0/Dockerfile

```diff
@@ -0,0 +1,84 @@
+FROM openjdk:8-jre-slim
```

Comment:
let's use jdk:11 - zk 3.6 works on java11

Reply:
Done, I build it locally and it runs on jdk 11